.travis.yml

```diff
@@ -8,16 +9,15 @@ matrix:
     - env: variant=test_release coverage=1 tests=1 toolset=gcc-coverage python=1
     - env: autotools=1 toolset=gcc
     - os: osx
-      osx_image: xcode7.3
+      osx_image: xcode8.3
```

Comment:
testing with the minimal version of Xcode is useful, unless you want to stay always with the latest version, is that the case?